### PR TITLE
French translation to "Advanced throwing" and "Advanced fatigue"

### PR DESCRIPTION
**Pull request by NorXAengell**
When merged this pull request will :
Add french translation to "Advanced throwing" and "Advanced fatigue"

### CODE REVIEW

**Comment by PabstMirror** [comment]
No changes on this branch, look like it's on your master branch
https://github.com/NorXAengell/ACE3/commit/b65217ebb776f056554d37b205e757ba899e7f36

**Comment by NorXAengell** [comment]
oh...Sorry -_-...How i can change the pull from the "trad" branch to my "master" ? Or i need to do a new pull from my master ?

**Comment by PabstMirror** [comment]
Either make this PR: https://github.com/acemod/ACE3/compare/master...NorXAengell:master
or merge master into your trad branch

**Comment by NorXAengell** [comment]
I don' fully understand how Git work but i think it's good now. Can you confirm ?
Thanks again for your help :).

**Comment by PabstMirror** [comment]
bon :+1:

**Comment by bux578** [comment]
Tagging qalganthe @alganthe

File: addons/advanced_fatigue/stringtable.xml [hunk 1]
```diff
@@ -1,57 +1,70 @@
...
     <Package name="Advanced_Fatigue">
         <Key ID="STR_ACE_Advanced_Fatigue_PerformanceFactor">
             <English>Performance Factor</English>
             <German>Leistungsfaktor</German>
+            <French>Facteur de performance</French>
         </Key>
         <Key ID="STR_ACE_Advanced_Fatigue_PerformanceFactor_Description">
             <English>Influences the overall performance of all players with no custom factor. Higher means better.</English>
             <German>Beinflusst die Leistungsfähigkeit aller Spieler ohne eigenen Leistungsfaktor. Ein höherer Wert bedeutet bessere Leistung.</German>
+            <French>Influence les performances générales de tout les joueurs sans facteur personalisé. Une valeur plus élevée implique de meilleur performance.</French>
```
Comment (alganthe) [inline addons/advanced_fatigue/stringtable.xml]: Influence les performances générales de tou`s` les joueurs sans facteurs `personnalisés`. Une valeur plus élevée implique de meilleur`es` performance`s`.

File: addons/advanced_fatigue/stringtable.xml [hunk 2]
```diff
@@ -1,57 +1,70 @@
...
         </Key>
         <Key ID="STR_ACE_Advanced_Fatigue_PerformanceFactor_EdenDescription">
             <English>Influences the overall performance of this unit. Higher means better.</English>
             <German>Beinflusst die Leistungsfähigkeit dieser Einheit. Ein höherer Wert bedeutet bessere Leistung.</German>
+            <French>Influence les performances générales de cette unité. Une valeur plus élevée implique de meilleur performance.</French>
```
Comment (alganthe) [inline addons/advanced_fatigue/stringtable.xml]: implique de meilleur`es` performance`s`

File: addons/advanced_fatigue/stringtable.xml [hunk 3]
```diff
@@ -1,57 +1,70 @@
...
         </Key>
         <Key ID="STR_ACE_Advanced_Fatigue_LoadFactor">
             <English>Load Factor</English>
             <German>Gewichtsfaktor</German>
+            <French>Facteur d'encombrement</French>
         </Key>
         <Key ID="STR_ACE_Advanced_Fatigue_LoadFactor_Description">
             <English>Increases or decreases how much weight influences the players performance. Zero means equipment weight has no performance influence.</English>
             <German>Erhöht oder verringert, wie viel Einfluss das Ausrüstungsgewicht auf die Leistung hat. Null heißt, dass es keinen Einfluss hat.</German>
+            <French>Augmente ou réduit l'influence qu'à le poids sur les performances des joueurs. Zéro implique que le poids de l'équipement n'a pas d'influence sur les performances.</French>
```
Comment (alganthe) [inline addons/advanced_fatigue/stringtable.xml]: qu'à le poids sur les performances 
devient
que le poids à sur les performances

File: addons/advanced_throwing/stringtable.xml [hunk 4]
```diff
@@ -4,86 +4,107 @@
...
         </Key>
         <Key ID="STR_ACE_Advanced_Throwing_Extend">
             <English>(Scroll) Extend</English>
             <Russian>(Скролл) Увеличить</Russian>
+            <French>(Molette souris) Etendre</French>
         </Key>
         <Key ID="STR_ACE_Advanced_Throwing_Cook">
             <English>(Click) Cook</English>
             <Russian>(Клик) Подготовить</Russian>
+            <French>(Click) Dégoupiller</French>
```
Comment (alganthe) [inline addons/advanced_throwing/stringtable.xml]: (Clique)

**Comment by NorXAengell** [comment]
Thx alganthe for the correction :). Correction done and by the way i have added others translations/corrections.

File: addons/dogtags/stringtable.xml [hunk 5]
```diff
@@ -6,36 +6,42 @@
...
             <Russian>Взять</Russian>
             <Czech>Vezmi</Czech>
+            <French>Prendre</French>
         </Key>
         <Key ID="STR_ACE_Dogtags_takeDogtagSuccess">
             <English>Dogtag taken from %1...</English>
             <Polish>Zabrałeś nieśmiertelnik %1...</Polish>
             <Russian>Жетон снят с %1...</Russian>
             <Czech>Sebral jsem známku od %1...</Czech>
+            <French>Plaque d'identification pris sur %1...</French>
```
Comment (alganthe) [inline addons/dogtags/stringtable.xml]: pris`e` sur

File: addons/dogtags/stringtable.xml [hunk 6]
```diff
@@ -6,36 +6,42 @@
...
         </Key>
         <Key ID="STR_ACE_Dogtags_dogtagAlreadyTaken">
             <English>Somebody else has already taken the dogtag...</English>
             <Polish>Ktoś już zabrał ten nieśmiertelnik...</Polish>
             <Russian>Кто-то уже забрал жетон...</Russian>
             <Czech>Někdo jiný už vzal identifikační známku...</Czech>
+            <French>Quelqu'un d'autre a déjâ pris les plaques d'identification...</French>
```
Comment (alganthe) [inline addons/dogtags/stringtable.xml]: déj`à`

File: addons/cookoff/stringtable.xml [hunk 7]
```diff
@@ -6,12 +6,14 @@
...
             <Czech>Povolit explozi munice</Czech>
             <Russian>Включить воспламенение</Russian>
+            <French>Activer </French>
         </Key>
         <Key ID="STR_ACE_CookOff_enable_tooltip">
             <English>Enables cook off and related vehicle destruction effects.</English>
             <German>Ermöglicht Cook-off und zugehörige Fahrzeug-Zerstörungseffekte.</German>
             <Czech>Povolí explozi munice a její následné ničivé efekty.</Czech>
             <Russian>Включает воспламенение и сопутствующие эффекты повреждения техники.</Russian>
+            <French>Active le et les effets de destructions assosicée aux vehicules.</French>
```
Comment (alganthe) [inline addons/cookoff/stringtable.xml]: Il manque un pan de phrase non ?
La traduction qui me semble correcte pour cook off est:
explosion par échauffement rapide.
Donc la phrase serait:
Active les explosions par échauffement rapide et les effets de destructions associés aux véhicules.

**Comment by PabstMirror** [comment]
Is this good to merge?

**Comment by PabstMirror** [comment]
Resolved merge conflicts
@NorXAengell @alganthe Is this ready?

**Comment by alganthe** [comment]
Looks good 👍

**Pull request merged**